### PR TITLE
Repair swagger

### PR DESCRIPTION
**Pull request by cmh2166**
Regenerating API code to work. Removed status from Resource model, since status isn't yet determined as something the API code needs to be aware of.

Dependencies (UUID library) are missing from this PR (are getting put into $GOPATH via `go get` and not into `dep ensure`) that need to be documented and added. We also have a dep / aws-sdk for go conflict. Opened issues #75 #74 for these, respectively.

### CODE REVIEW

**Comment by cmh2166** [comment]
Also moves up in priority #42 and #73 (not just dep version, but go-swagger version)

File: swagger.json [hunk 1]
```diff
@@ -33,21 +33,7 @@
...
-                "state" : {
-                  "type" : "string",
-                  "example" : "in progress"
-                },
-                "request_id" : {
-                  "type" : "string",
-                  "example" : "0a72c00a-1332-4730-b19c-9131c3bcb57f"
-                }
-              }
+              "$ref" : "#/definitions/Resource"
```
Comment (jcoyne) [inline swagger.json]: I don't agree that this is the correct response for a create. As documented on the whiteboard the response should be a RequestID, a ResourceID and a RequestStatus.   I don't think it makes sense to send them back the resource they just sent us.

**Comment by jcoyne** [comment]
I'm curious to know what you mean about the UUID library being missing. I see it in the lockfile right here: https://github.com/sul-dlss-labs/taco/blob/master/Gopkg.lock#L156-L159

**Comment by cmh2166** [comment]
It wasn’t in the toml file for me to then ensure off of. Can confirm again when at a computer.

**Comment by jcoyne** [comment]
@cmh2166 are we certain it needs to be in the toml file? I thought that was just for the ones it can't introspect.  The true list of dependencies is in the .lock file.

**Comment by cmh2166** [comment]
Check the docs around `dep ensure -update`: https://golang.github.io/dep/docs/daily-dep.html

**Comment by jcoyne** [comment]
@cmh2166 yeah on that page I'm seeing:
> Of course, given this model, you don't have to use dep ensure -add to add new dependencies - you can also just add an appropriate import statement in your code, then run dep ensure. However, this approach doesn't always play nicely with goimports, and also won't append a [[constraint]] into Gopkg.toml. Still, it can be useful at times, often for rapid iteration and off-the-cuff experimenting.

So I'm wondering what the drawback to this approach is. This also seems to be what one of the maintainers recommends as her preferred workflow: https://github.com/golang/dep/issues/376#issuecomment-293964655

**Comment by cmh2166** [comment]
@jcoyne i'm not commenting on the approach against, i'm saying we don't do that cleanly right now (hence your original PR adding all this dynamo stuff not working when I pulled down **and only ran** `dep ensure`).

Regardless, that is ticketed and does not stop this PR.

**Comment by cmh2166** [comment]
Closing ticket given recent updates. Doesn't close issues related since those are still undetermined (and the code / spec still need cleaning up).

**Pull request closed without merging**